Doc/reference/compound_stmts.rst

```diff
@@ -1336,6 +1338,10 @@ enables postponed evaluation.  Otherwise, they are evaluated when the function
 definition is executed.  In this case annotations may be evaluated in
 a different order than they appear in the source code.
 
+.. versionchanged:: 3.11
+   Parameters of the form "``*identifier``" may have an annotation
+   "``: *expression``". See :pep:`646` for details.
```

Comment:
It's fine to link to PEP-646 from these docs, as PEP-646 explains the motivation and history behind this change in a complete way. But nowadays we prefer to have all relevant details in the CPython docs, and for PEPs to be seen as historical documents that are generally not meant to be updated after they're accepted. So I'd prefer us to avoid, if possible, the implication that there are any details in PEP-646 that aren't included in the CPython docs, because ideally there shouldn't be any.

Reply:
(I know lots of other bits of the docs say things like "See PEP XXX for details", but this is something we're trying to slowly move away from!)

Reply:
I think a versionchanged note is a good place to link a PEP, because it provides background and context for the change. Perhaps we can simply:
```suggestion
   "``: *expression``". See :pep:`646`.
```
to remove the implication that the PEP has more information specifying the behavior.